jump-core/src/main/java/com/bitdecay/jump/collision/CollisionUtilities.java

```diff
@@ -24,10 +24,11 @@ public static boolean canTileCollisionCanBeSkipped(TileBody tileBody, Manifold m
             return true;
         }
 
-        if (manifold.distance > 0 && lastPosition < resolutionPosition) {
-            // The actor was already inside the tileBody before it moved, so we can skip it.
-            return true;
-        }
+        // TODO: This needs to be revisited to see if it is useful or necessary. Currently it's just causing bugs
+//        if (manifold.distance > 0 && lastPosition < resolutionPosition) {
+//            // The actor was already inside the tileBody before it moved, so we can skip it.
+//            return true;
+//        }
```

Comment:
Leaving this commented out because it may be something we want to re-implement once we find a better approach.